XMLHttpRequest/firing-events-http-content-length.html

```diff
@@ -17,20 +17,20 @@
         xhr.onprogress = function(pe) {
           test.step(function() {
             if(pe.type == "progress") {
-              assert_true(pe.loaded >= 0, "loaded is initialize to the number of HTTP entity body bytes transferred.");
-              assert_true(pe.lengthComputable, "lengthComputable is true.");
-              assert_not_equals(pe.total, 0, "total is not zero.");
+              assert_greater_than_equal(pe.loaded, 0, "loaded");
```

Comment:
I simplified the descriptions because the failure messages actually make more sense when just the actual value is described, as opposed to what it should be.